docs/docs/noir/concepts/functions.md

```diff
@@ -34,6 +34,15 @@ is pre-pended with a colon and the parameter type. Multiple parameters are separ
 fn foo(x : Field, y : Field){}
 ```
 
+You can use an underscore `_` as a parameter name when you don't need to use the parameter in the function body. This is useful when you need to satisfy a function signature but don't need to use all the parameters:
+
+```rust
+fn foo(_ : Field, y : Field) {
+    // Only using y parameter
+    y + 1
+}
```

Comment:
This function isn't valid noir as it's returning a value while the return type of the function is `()`.